src/component/Detail.js

```diff
@@ -0,0 +1,65 @@
+import './home.css';
+import React, { useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useSelector, useDispatch } from 'react-redux';
+import { Card } from 'react-bootstrap';
+import { IoIosArrowBack } from 'react-icons/io';
+import { NavLink } from 'react-router-dom';
+import { fetchDetail } from '../redux/cuisine/detailSlice';
+
+export default function Detail() {
```

Comment:
Nice job so far😉. Let's try to ensure our project matches the given design. take note of the highlighted components. also, do well to stay consistent with the color as done in design 😉. You should use the same lighter shade of blue used on your home page i.e `#6b8ecf` to keep the colors consistent as seen in the behance design.  , instead of making use of white. do well to implement that.

<img width="295" alt="image" src="https://github.com/MayPyone/React_Capstone_Project/assets/52934128/0d161a90-4050-4577-a8ad-0e1b36bb94bd">